File: src/main/java/com/akagiyui/drive/model/response/LoginResponse.java
package com.akagiyui.drive.model.response;

import lombok.AllArgsConstructor;
import lombok.Data;

/**
 * 登录响应
 * @author AkagiYui
 */
@Data
@AllArgsConstructor
public class LoginResponse {
    /**
     * token
     */
    private String token;
    /**
     * 刷新token
     */
    private String refreshToken; // TODO 刷新token
}

File: src/main/kotlin/com/akagiyui/drive/model/response/LoginResponse.kt
package com.akagiyui.drive.model.response

/**
 * 登录响应
 * @author AkagiYui
 */
data class LoginResponse(
    /**
     * token
     */
    val token: String,
    /**
     * 刷新token
     */
    val refreshToken: String? = null, // TODO 刷新token
)
